build.gradle

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2017 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+plugins {
+    id "nebula.plugin-plugin" version "5.1.1"
+    id 'com.gradle.plugin-publish' version '0.9.7'
+}
+
+version '0.1'
+
+apply plugin: 'groovy'
+
+apply from: 'gradle/dependencies.gradle'
+
+contacts {
+    'perikov.igor@gmail.com' {
+        moniker 'Igor Perikov'
+        github 'IgorPerikov'
+    }
+}
+
+pluginBundle {
+    website = 'http://www.hollow.how'
+    vcsUrl = 'https://github.com/nebula-plugins/hollow-plugin'
+    description = 'This plugin provides a task for generating hollow consumer api.'
+    tags = ['hollow', 'nebula']
+
+    plugins {
+        hollowPlugin {
+            id = 'nebula.hollow-plugin'
```

Comment:
yes, that's the 1st part of total work, 2nd is to rename `nebula.hollow-plugin.properties` to `nebula.hollow.properties`